styles.css

```diff
@@ -194,11 +194,10 @@ ul {
   padding: 10px;
   background: #ff6b00;
   transition: all 1s linear;
-  
 }
 
 .button1:hover {
-  background-color:#fed8b1;
+  background-color: #fed8b1;
```

Comment:
- [OPTIONAL] Great job adding hover here 👌, I suggest adding a font color `color: #F55800` to the button to make it look like a design.